spec/registry/spec.md

```diff
@@ -62,6 +62,58 @@ themselves discoverable via a [[ref:Gateway]]'s API.
 | Web App                 | https://schema.org/WebApplication         | 6            |
 | Financial Institution   | https://schema.org/FinancialService       | 7            |
 
+### Interoperable DID Methods
+
+As an **OPTIONAL** extension, some existing DID methods can leverage `did:dht` to broaden their feature set. This registry serves
+to define such DID methods that are interoperable with `did:dht`. To enter into the registry each DID method ****MUST**** define
+sections on _publishing_ and _resolving_ interoperable DIDs.
+
+#### did:key
+
+DID DHT is interoperable with the [[ref:DID Key method]] when using [[ref:Ed25519]] keys represented as JWKs [[spec:RFC7517]].
+
+##### Publishing
+
+To publish a [[ref:DID Key]] to the DHT, the process is as follows:
+
+1. Verify the `did:key` value begins with the prefix `z6Mk`.
+2. Decode the [[ref:Ed25519]] key in the `did:key` identifier, and re-encode it using [[ref:z-base-32]].
+3. Expand the `did:key` using the [process outlined in the did:key spec](https://w3c-ccg.github.io/did-method-key/#read), 
+with `options.publicKeyFormat` set to `JsonWebKey2020`.
+4. Optionally, amend the [[ref:DID Document]] with additional properties (Verification Methods, Services, etc.).
+5. Follow steps 3 onward in as outlined in the [create section of the spec](../index.html#create), using the identifier from step 2.
```

Comment:
We'll want to modify this slightly to:

1. Use the different prefix for a `did:dht` Ed25519 key (that just makes sure we bind the Ed25519 key to a resolution network -- `did:dht`). We can register the Multikey prefix pretty quickly to do that.
2. did:key can take an argument and you can get your keys back as either Multikey or JsonWebKey, so no need to be specific here. `JsonWebKey2020` is definitely out of date and is not going to survive standardization. We should be using `JsonWebKey` anywhere that we were using `JsonWebKey2020` before.
3. If you wanted the `did:key` to be encoded in z-base-32, we could do that... though I don't think there's a Multibase encoding for that yet? If there is interest there, we could look into that (this is one of the reasons Multibase exists -- because the base-encoding you use is largely driven by what the protocols are optimized for -- humans, ASCII, glyph compression, or some combination of the previous).

Reply:
agree with 1 + 2 will update when the did:key spec is updated ... (https://github.com/TBD54566975/did-dht-method/issues/57)

for 3 I think this is a good idea as it reduces the need to re-encode/decode here for resolvers...but that raises a question. Is it preferable to reduce the impact on existing implementations (i.e. not need to support z-base-32) and just keep the translation a function of resolvers/registrars?

Reply:
re: 3 -- yes, it's a good question. I haven't had enough time to think about it in detail... my gut tells me to not add the z-base-32 requirement for `did:key` implementations... but then again, it's a fairly trivial piece of code to write (it's just [general base-encoding](https://w3c.github.io/vc-data-integrity/#base-encode) w/ a different alphabet).